code/modules/parallax/parallax_render_sources/parallax_render_source_group_parent.dm

```diff
@@ -0,0 +1,147 @@
+ABSTRACT_TYPE(/datum/parallax_render_source_group)
+/**
+ *	The parent type for parallax render source groups, which govern which parallax render sources should be shown to a client
+ *	in a given area or z-level. They also allow for the efficient manipulation of parallax render sources and layers within a
+ *	specified z-level or area, permitting more granular control over the appearance of parallax render sources.
+ */
+/datum/parallax_render_source_group
+	/// An associative list of parallax render source types and the instance of that type belonging to this render source group.
+	var/list/atom/movable/screen/parallax_render_source/parallax_render_source_types_and_sources
+	/// A list of default parallax layer render source types that should be instantiated on New().
+	var/list/parallax_render_source_types = list()
```

Comment:
do this in New also

Reply:
`parallax_render_source_types` is the list populated with, as one could guess, default parallax render sources in the various subtypes of `/datum/parallax_render_source_group` - setting it to `list()` in `New()` would erase those default render sources.